src/test/resources/testcontainers.properties

```diff
@@ -1,2 +1,2 @@
-# Digest of image testcontainersofficial/ryuk:latest for linux/amd64 from tags list on https://hub.docker.com/r/testcontainersofficial/ryuk
-ryuk.container.image=testcontainersofficial/ryuk@sha256:8fdd715bf34975d088273bcea497fe048f9b768ba4c7fc690ab6e4ddc995e317
+# arm64/amd64 friendly ryuk image
+ryuk.container.image=testcontainers/ryuk:0.3.3
```

Comment:
Is there any difference between `testcontainers` and `testcontainersofficial`? They both seem up to date 🤔

Reply:
afaict, the difference is that the former vends multi platform images whereas the latter does not...